legofy/palettes.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+
+"""
+legofy.palettes
+---------------
+
+This module contains the `lego` palette mappings.
+
+Color mapping source;
+ - http://www.brickjournal.com/files/PDFs/2010LEGOcolorpalette.pdf
+
+
+    USAGE:
+    $ legofy.palettes.legos
+
+See README for project details.
+"""
+
+
+LEGOS = {
+    'solid': {

```

Comment:
How about we format the data so that it is instantly consummable?
```
PALETTES = {
     'solid' : [
        0xfe, 0xc4, 0x01, #024
        0xe7, 0x64, 0x19, #106
       ...
    ]
}
```

This would reduce the amount of code and transformations needed.